tokio/src/task/blocking.rs

```diff
@@ -102,8 +102,17 @@ cfg_rt! {
     /// their own. If you want to spawn an ordinary thread, you should use
     /// [`thread::spawn`] instead.
     ///
-    /// Closures spawned using `spawn_blocking` cannot be cancelled. When you shut
-    /// down the executor, it will wait indefinitely for all blocking operations to
+    /// Closures spawned using `spawn_blocking` cannot be cancelled abruptly; there
+    /// is no standard low level API to cause a thread to stop running.  However,
+    /// a useful pattern is to pass some form of "cancellation token" into
+    /// the thread.  This could be an [`AtomicBool`], and inner loops in the
+    /// thread check the boolean periodically.  Another approach is to
```

Comment:
```suggestion
    /// the thread.  This could be an [`AtomicBool`] that the task checks periodically.  Another approach is to
```

Reply:
Thanks, agree that phrasing is clearer.  Fixed!